models/dojo.py

```diff
@@ -0,0 +1,277 @@
+import random
+
+from models.room.office_space import Office
+from models.room.living_space import LivingSpace
+from models.person.fellow import Fellow
+from models.person.staff import Staff
+
+
+class Dojo(object):
+    """The Dojo class."""
+
+    office_rooms = {}
+    living_rooms = {}
+    persons = {}
+    divider = ("\n{}\n".format("-" * 30))
+
+    def __init__(self):
+        pass
+
+    def create_room(self, arg):
+        """The function creates a new room."""
+        room_type = arg["<room_type>"]
+        room_names = arg["<room_name>"]
+        for room_name in room_names:
+            if room_name == ('office' or 'livingspace'):
+                raise Exception("This is not a valid room name")
+            else:
+                if room_type.upper() == "OFFICE":
+                    self.office_rooms[room_name] = Office(room_name)
+
+                elif room_type.upper() == "LIVINGSPACE":
+                    self.living_rooms[room_name] = LivingSpace(room_name)
+
+                else:
+                    raise Exception("Invalid Room Type.Must be office or living")
+
+                Prefix = ("A" if room_type.upper() == "LIVINGSPACE" else "An")
```

Comment:
variable naming in python are usually lowercased, and snake case
save for classes that are capitalized and camel
use `prefix` instead